src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/LoggerWrapper.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.CommonLanguageServerProtocol.Framework;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer;
+
+public class LoggerWrapper : ILspLogger, ILogger
```

Comment:
Consider using the word "Adapter" instead of "Wrapper". That does a slightly better job at describing the class's purpose rather than what it does.